### ISSUE TITLE
Bug Report: Changing IP no longer possible on debian host

### ISSUE THREAD

**Issue by Buster14** [home-assistant/supervised-installer#246]
### OS Version

Debian GNU/Linux 11 (bullseye)

### System Information

Linux gowongan 5.10.0-18-amd64 #1 SMP Debian 5.10.140-1 (2022-09-02) x86_64 GNU/Linux

### What happened?

Hi, I have an Asus v161 + debian running with this script. 

Problem is My debian wont retain its static ip address change after reboot. Tried everything i found online.
I also run frigate, mqtt and nativefier'ed lovelace webpage, so everytime the ip changes i have to change the config as well for those 3 to match the ip.
I'm pretty sure it's still dhcp cause yesterday i accidentaly pressed the gnome network manager on the bottom right and suddenly the ip changes again. (It's supposed to just disconnect and reconnect it)

What i have tried :
1. change /etc/network/interfaces
2. then i found out it has sources interfaces.d/setup so i change it as well.
3. turns out there is interfaces.real so i changed it too..
4. systemctl restart NetworkManager,reboot, 
but ip a still showing the old ip. 

Does it have anything to do with the HA container?
At least is there any way to make the IP static?

### Machine Type

generic-x86-64

### ADR

- [X] I have read through the [ADR](https://github.com/home-assistant/architecture/blob/master/adr/0014-home-assistant-supervised.md) and have confirmed that my system is compliant with the requirements
- [X] I understand that if my system is found to not be compliant, my issue will be closed immediately without further investigation

### Code of Conduct

- [X] I have read and agree to the [Home Assistant Code of Conduct](https://www.home-assistant.io/code_of_conduct/)

**Comment by ikifar2012** [comment]
The official way to set a static IP is to use the Home Assistant UI
(Settings > System > Network)

[![Open your Home Assistant instance and manage your systems network configuration.](https://my.home-assistant.io/badges/network.svg)](https://my.home-assistant.io/redirect/network/)

**Comment by Buster14** [comment]
So after installing the script, there's no way to change the  debian host IP other than from HA?
Can you confirm this?
I just checked and the HA IP is already set on static.
It's kinda strange that disabling and enabling the ethernet on debian gnome network manager would change the IP of the host (and thus the containers).

**Comment by Buster14** [comment]
Hi, I think I might have figured out the root cause..
My debian installation username has same username as HA. The debian username is non root.
Can you confirm if this might cause issues?
I will reinstall debian with different username then..

**Comment by ikifar2012** [comment]
I have never installed supervised on a Desktop Environment, I have a feeling your DE is interfering with HA supervised

**Comment by Buster14** [comment]
For Audio looks like it's indeed an existing issue, I managed to solved it using addon as described here 
https://community.home-assistant.io/t/solved-hassio-audio-is-in-boot-loop-and-spams-my-syslog-with-errors/282538/106

For IP, I just found out that my HAOS installation doesn't have gateway yet, I have set it and reinstall my debian, will see if it solves the issue.
As long as it doesn't change ip after reboot then i wont need to mess with the ip.

Yeah I need DE cause I installed on asus v161 to act as lovelace dashboard as well.
